CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * `jj split` supports the `--interactive` flag.
 
+* `jj commit` accepts an optional list of paths indicating a subset of files to
+  commit.
```

Comment:
I thought about this only while reviewing #2133, but it applies here too:

IMO, the command name is `commit` to make it discoverable for git/hg users, but I think we should still avoid saying "to commit". The command's help text for `jj commit` says "Update the description and create a new change on top.", so I think it would make sense for this to say something like "... to include in the first commit" or "... to include in the parent commit".

Reply:
@martinvonz I changed it to "... to include in the parent revision", since it seems like we use "revision" as a noun more often than we use "commit" in the changelog. Perhaps we should also change this instance in the changelog?

https://github.com/martinvonz/jj/blob/9d8be29d94b1c2e9959be76cb4cbfa17f890bfa1/CHANGELOG.md#L21

Reply:
Yes, sounds good.